Sources/PT.PM.Common/Nodes/Comment.cs

```diff
@@ -0,0 +1,19 @@
+﻿using MessagePack;
+
+namespace PT.PM.Common.Nodes.Tokens.Literals
+{
+    [MessagePackObject]
+    public class Comment : Ust
```

Comment:
`Comment` is still in `Literals` namespace despite not inheriting from `UstLiteral`?

Reply:
Actually, comments are hidden tokens, they can be completely skipped without influence compilation. In Roslyn, they are named as [syntax trivia](https://github.com/dotnet/roslyn/wiki/Roslyn-Overview#syntax-trivia) that are not inherited from any ast class. So, comments are even not `Ust`, but I left it by such way, because of some compatibility reasons.

Also, we have another meaning of literals - they can be placed to any expression.